packages/dynamic-forms-ionic/styles/_defaults.scss

```
@@ -0,0 +1,67 @@
+// ============================================
+// IONIC FORM DEFAULTS
+// ============================================
+// Opinionated host-framework defaults for Ionic apps.
+//
```

Stylelint (17.11.1)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)